### ISSUE TITLE
Suggestion: Improve challenges confirmation (Please confirm that you want to replace your current challenges with new one.)

### ISSUE THREAD

**Issue by IAISI** [BentoBoxWorld/Challenges#329]
### Is your feature request related to a problem?

When using /bsbadmin challenges to import/manage challenges player is sometimes asked to confirm action with message:

Please confirm that you want to replace your current challenges with new one.

In a way this message is not really intuitive, I'm a bit ashamed to admit this but personally it took me day to figure out I just have to type "confirm" in change to confirm it.

### Describe the solution you'd like.

Maybe append "Please type "confirm" in chat to confirm this action." to the message?

### Describe alternatives you've considered.

None.

### Agreements

- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have ensured the feature I'm requesting isn't already in the latest supported BentoBox build or Addons.

**Comment by tastybento** [comment]
Agreed, this trips people up. The confirmation conversation only accepts specific words (the locale's confirm/deny strings — "confirm", "yes", "cancel", …) but the prompt never tells you that; it just asks the question and silently rejects anything else.

Fix: have `ConversationUtils.createConfirmation()` append a standard instruction line to every confirmation question — "Type 'confirm' to proceed or 'cancel' to abort" — driven by a locale string so translations can match their own confirm/deny words. One change, and it improves every confirmation (import, reset, wipe, …).

Verifiable with a JUnit test on the prompt text, plus a quick in-game check on the import confirmation. Marking P2.

**Comment by tastybento** [comment]
Done in #415. The confirmation prompt now appends a standard instruction line — "Type 'confirm' in chat to proceed, or 'cancel' to abort." — driven by a new `confirm-instruction` locale string, so it applies to every confirmation (import, reset, wipe, …) and translators can word it (and their own confirm/deny terms) however they like. No more guessing that you have to type the word in chat.
